learn/cookbooks/docker.md

```diff
@@ -0,0 +1,123 @@
+# Using Meilisearch with Docker
+
+Docker is a tool that packages applications into containers portable across a wide range of environments. When using Docker for development, we recommend following [the official instructions to install Docker Desktop](https://docs.docker.com/get-docker/).
+
+In this guide you will learn how to use Docker to download and run Meilisearch, configure your instance, and manage your Meilisearch data.
+
+## Download Meilisearch with Docker
+
+Docker containers are distributed in images. To use Meilisearch, use the `docker pull` command to download a Meilisearch image:
+
+```sh
+docker pull getmeili/meilisearch:v0.28
+```
+
+Meilisearch deploys a new Docker image for every new release. All images are tagged with the release version—indicated in the example above by the text following the `:`. You can see [the full list of available Meilisearch Docker images](https://hub.docker.com/r/getmeili/meilisearch/tags#!) on Docker Hub.
+
+::: warning
+The `latest` tag will always download the most recent Meilisearch release. Meilisearch advises against using it, as it might result in different machines running different images if significant time passes between setting up each one of them.
+:::
+
+## Run Meilisearch
```

Comment:
Can I push back on this? I just feel that being this specific doesn't add much—technically, yes, we're not giving instructions about running the Meilisearch binary, but that's what the user cares about, no?

Also, Docker terminology is confusing. I want to avoid the discussion on whether we are running the image or the container.

Reply:
I was inspired to make this suggestion by seeing that you titled the first section `Download Meilisearch with Docker`. I think that's a great choice that shows a lot of foresight, because when a user searches "Download Meilisearch" and it returns this heading, we want them to understand what they're clicking on. I think you can apply the same logic to this heading: we probably wouldn't want someone searching "run meilisearch" (or "install meilisearch" or any other term that we may define as a synonym) to find this result.

I stand by my suggestion (amended in response to your second point) but leave the decision up to you.

```suggestion
## Run Meilisearch with Docker
```

Reply:
Ok, sounds good. Seeing your second suggestion I realise my problem was more with terminology than your actual main point.